user_docs/en/userGuide.t2t

```diff
@@ -2471,6 +2471,9 @@ Following are the command line options for NVDA:
 | None | --no-sr-flag  | Don't change the global system screen reader flag |
 | None | --install | Installs NVDA (starting the newly installed copy) |
 | None | --install-silent | Silently installs NVDA (does not start the newly installed copy) |
+| None | --create-portable | Creates a portable copy of NVDA (starting the newly created copy) |
+| None | --create-portable-silent | Creates a portable copy of NVDA (does not start the newly installed copy) |
+| None | portable-path=PORTABLEPATH | The path where a portable copy will be created |

```

Comment:
Should this argument start with `--` ?
Also could you document the dependencies between the arguments?